experimental/dds/tree2/src/feature-libraries/nodeIdentifier.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { StableId } from "@fluidframework/runtime-definitions";
+import { Brand, brand } from "../util";
+import { TreeSchemaIdentifier, ValueSchema } from "../core";
+import {
+	FieldEditor,
+	GlobalFieldSchema,
+	Multiplicity,
+	SchemaBuilder,
+	SchemaLibrary,
+	TreeSchema,
+} from "./modular-schema";
+import { BrandedFieldKind, nodeIdentifier } from "./defaultFieldKinds";
+
+/**
+ * The primitive type used to identify nodes in the tree.
+ * @alpha
+ */
+export type NodeIdentifier = Brand<StableId, "Node Identifier">;
+
+/**
+ * Get the schema for working with {@link NodeIdentifier}s in a tree. Node identifiers are added to nodes via a
+ * global field.
```

Comment:
I prefer more semantic line breaks. For example, break the line between the two sentences instead of in the middle of the second one. This has better interop with git, our code review tools, diffs etc. I also find it easier to read, and it means you don't have to deal with the two spaces after a period debate.
```suggestion
 * Get the schema for working with {@link NodeIdentifier}s in a tree.
 * Node identifiers are added to nodes via a global field.
```

There are several other places this could be applied.

Reply:
Sure, that makes sense. I changed this one (and another in this PR) to break over periods and commas. I don't know if I agree it's best to break after _every_ period (maybe I do?) but in the future it'd be fun to formalize an actual set of rules, update our existing docs, and see how far we can push prettier to enforce it.